pkg/cloud/services/ec2/securitygroups.go

```diff
@@ -47,6 +46,15 @@ const (
 	IPProtocolICMPv6 = "58"
 )
 
+const (
+	// user set specify cni
+	clusterSetCNI = "cluster.k8s.io/network-cni"
+
+	// the amazon native network solution
+	// https://github.com/aws/amazon-vpc-cni-k8s
+	amazonVPCCNI = "amazon-vpc-cni-k8s"
```

Comment:
Would like to follow K8s convention for CamelCase on field values and use `AmazonVPC`

Reply:
> Would like to follow K8s convention for CamelCase on field values and use `AmazonVPC`?

copy that.